docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+.. post:: October 9, 2017
+   :tags: newsletter
+
+
+#####################################
+Write the Docs Newsletter – October
```

Comment:
I like including the year in the title for consistency.

Reply:
👍